docs/pages/production-configuration.rst

```diff
@@ -13,17 +13,23 @@ Persistence
 All data the node produces is stored on disk in the configured data directory (`datadir`). It is recommended to backup
 everything in that directory.
 
-The private keys are stored in a storage backend. Currently 2 options are available:
+The private keys are stored in a storage backend. Currently 2 options are available.
 
-* Vault:
-This storage backend is the recommended way of storing secrets. It uses the Vault KV version 1 store:
-https://www.vaultproject.io/docs/secrets/kv/kv-v1. All private keys are stored under the path `<prefix>/nuts-private-keys/*`.
-The default prefix `kv` and can be configured using the `crypto.vault.pathprefix` option.
+Vault
+^^^^^
+
+This storage backend is the recommended way of storing secrets. It uses the `Vault KV version 1 store <https://www.vaultproject.io/docs/secrets/kv/kv-v1>`_.
+The prefix defaults to `kv` and can be configured using the `crypto.vault.pathprefix` option.
+There needs to be a KV Secrets Engine (v1) enabled under this prefix path.
+
+All private keys are stored under the path `<prefix>/nuts-private-keys/*`.
 Each key is stored under the kid, resulting in a full key path like `kv/nuts-private-keys/did:nuts:123#abc`.
-A Vault token must be provided by either configuring it using the config `crypto.vault.token` or setting the VAULT_TOKEN environment variable.
+A Vault token must be provided by either configuring it using the `crypto.vault.token` configuration option.
```

Comment:
The word `either` is no longer needed. Does `VAULT_TOKEN` not work?

Reply:
I assumed it was an incorrect example on how to configure it using the Nuts node's env options. But it's actually from the vault client lib itself, so it was correct. I reverted it.

Although I'm not sure whether we'd want people configuring the vault client using env options parsed by the client lib instead of them being passed through the Nuts node config, since we then lose control over it. But I guess OK for now.